src/makeRobotAccountant.js

```diff
@@ -6,7 +6,27 @@
  */
 
 function makeRobotAccountant() {
-  // write code here
+  let counter = 0;
+  let isProblematic = false;
```

Comment:
```suggestion
  let isProblematic = false;
```
Тебе не нужна эта переменная